shanoir-ng-ms-common/src/main/java/org/shanoir/ng/utils/Range.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Shanoir NG - Import, manage and share neuroimaging data
+ * Copyright (C) 2009-2019 Inria - https://www.inria.fr/
+ * Contact us on https://project.inria.fr/shanoir/
+ * 
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+package org.shanoir.ng.utils;
+
+public class Range<T> {
```

Comment:
A quoi sert cette classe ?

Reply:
In the solr search we can search for datasets that numerical metadata values are within a range. For instance we want magnetic field strength to be between 2 and 5.
In this case I find it usefull for making the code and the transferred json objects tidier. Some other implementations exist which can feature an includes() method to check if a value is within the range or the management of including/excluding endpoints.